dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -81,6 +81,8 @@ UNEXPORTED_VARIABLES+=('PORT_LINUX' 'PORT_DARWIN')
 UNEXPORTED_VARIABLES+=('PORT[ ?=:]' 'PORT$')
 
 EXPORTED_VARIABLES_ONLY_IN_VARS=()
+EXPORTED_VARIABLES_ONLY_IN_VARS+=('CPU_ARCH')
+EXPORTED_VARIABLES_ONLY_IN_VARS+=('CPU_FAM CPU_FAM$')
```

Comment:
```suggestion
EXPORTED_VARIABLES_ONLY_IN_VARS+=('CPU_FAM$')
```

Reply:
We need both so there is not `export CPU_FAM = xxx` and not `export CPU_FAM`

Reply:
Then only CPU_FAM is needed, no ?

Reply:
It will match `CPU_FAMILY` if doing so, yes the arbitrary KINETIS names are annoying